src/pytest_workflow/plugin.py

```diff
@@ -339,7 +346,10 @@ def queue_workflow(self):
 
         # Copy the project directory to the temporary directory using pytest's
         # rootdir.
-        shutil.copytree(str(self.config.rootdir), str(tempdir))
+        if self.config.getoption("symlink"):
+            link_tree(Path(str(self.config.rootdir)), tempdir)
```

Comment:
What is the type of self.config.rootdir that you have to cast it twice?

Reply:
some `py` library `LocalPath` thingy. So I convert it to string and then use pathlib instead.